pkg/controller/operandregistry/operandregistry_controller.go

```diff
@@ -51,6 +54,11 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
+	// Add extenal api scheme
```

Comment:
@DanielXLee Do we need to add the external api scheme here?
I think https://github.com/IBM/operand-deployment-lifecycle-manager/blob/master/pkg/apis/addtoscheme_operator_v1alpha1.go#L28 is enough to use OLM api.

Reply:
you are right, I'll remove it